pkgs/games/heroic/fhsenv.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, buildFHSUserEnv
+, heroic-unwrapped
+}:
+
+buildFHSUserEnv {
+  name = "heroic";
+
+  runScript = "heroic";
+
+  targetPkgs = pkgs: with pkgs; [
+    heroic-unwrapped
+
+    gnugrep
+    gnused
+    gnutar
```

Comment:
Some of these are already included by default https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix#L45-L48

Please remove them to keep the list shorter.

Reply:
Fixed.